### DISCUSSION TITLE
Création page ou section imperméabilisation

### DISCUSSION THREAD

**Discussion by inesdartiguenave** [MTES-MCT/sparte#461] [Idées]
Questions : 

- est ce qu'il y a forcément correspondance entre la prod et l'export ? Soit besoin d'avoir les graphiques d'imperméabilisation sur la plateforme pour les propose dans l'export de rapport local ? 

- Si oui, où proposer l'imperméabilisation ? Une section de l'onglet artificialisation ou découvrir l'OCS GE, ou un nouvel onglet ?

**Comment by Phil-Loriot** [comment]
et proposer exactement les mêmes graphiques et tableaux que pour l'artif mais uniquement avec l'imperméabilisation (CS1.1)

**Comment by alexisig** [comment]
> Pour l'encadré en tête d'onglet, mettre : 
> Le [Décret n° 2023-1096 du 27 novembre 2023 relatif à l'évaluation et au suivi de l'artificialisation des sols](https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000048465959) précise que le rapport relatif à l'artificialisation des sols prévu à l'article L. 2231-1 présente, pour les années civiles sur lesquelles il porte et au moins tous les trois ans, les indicateurs et données suivants :
> « 1° La consommation des espaces naturels, agricoles et forestiers, exprimée en nombre d'hectares, le cas échéant en la différenciant entre ces types d'espaces, et en pourcentage au regard de la superficie du territoire couvert. Sur le même territoire, le rapport peut préciser également la transformation effective d'espaces urbanisés ou construits en espaces naturels, agricoles et forestiers du fait d'une renaturation ; 
> « 2° Le solde entre les surfaces artificialisées et les surfaces désartificialisées, telles que définies dans la nomenclature annexée à l'[article R. 101-1 du code de l'urbanisme ](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006074075&idArticle=LEGIARTI000045729062&dateTexte=&categorieLien=cid); 
> **« 3° Les surfaces dont les sols ont été rendus imperméables, au sens des 1° et 2° de la nomenclature annexée à l'[article R. 101-1 du code de l'urbanisme ](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006074075&idArticle=LEGIARTI000045729062&dateTexte=&categorieLien=cid);**
> « 4° L'évaluation du respect des objectifs de réduction de la consommation d'espaces naturels, agricoles et forestiers et de lutte contre l'artificialisation des sols fixés dans les documents de planification et d'urbanisme. Les documents de planification sont ceux énumérés au [III de l'article R. 101-1 du code de l'urbanisme](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006074075&idArticle=LEGIARTI000045729062&dateTexte=&categorieLien=cid). 
> 
> L’imperméabilisation des sols est donc définie comme :
> 1° Surfaces dont les sols sont imperméabilisés en raison du bâti (constructions, aménagements, ouvrages ou installations).
> 2° Surfaces dont les sols sont imperméabilisés en raison d'un revêtement (artificiel, asphalté, bétonné, couvert de pavés ou de dalles).
> 
> Au niveau national, l’imperméabilisation est mesurée par l'occupation des sols à grande échelle (OCS GE), en cours d'élaboration, dont la production sera engagée sur l'ensemble du territoire national d'ici mi 2025.
> 
> Ajouter la matrice avec juste les 2 premières colonnes en rouge 

 _Originally posted by @Phil-Loriot in [#458](https://github.com/MTES-MCT/sparte/issues/458#issuecomment-2194156282)_
